README.md

```diff
@@ -43,27 +43,38 @@ If you have gone through the manual installation, you have the opportunity to ch
 Go to your Steam library -> right click the game -> Properties -> and edit the launch options to:
 
     thcrap_proton -- %command%
-   
+
 In case you have put your script outside `/usr/local/bin/`, you'll have to provide the full path:
-   
+
     /path/to/thcrap_proton -- %command%
 
 This is the base command, which will run the game with the default config.
 
 To change the config file loaded by thcrap, use the `-c` flag.
 
 To enable vpatch for that game, include the `-v` flag.
-   
+
 **Note that this script does not install vpatch on it's own.**
 
-So, if I wanted to run the game with vpatch and Brazilian Portuguese translations, the command would look like this:
-       
-    thcrap_proton -v -c pt-br.js -- %command%
+To enable thprac for that game, use the `-p` flag to specify thprac executable.
+Upon launching thprac will ask whether to apply to the ongoing game, if running game
+in full screen mode you may need to minimize the game to see the dialog box.
+To [change thprac language](https://github.com/touhouworldcup/thprac?tab=readme-ov-file#how-do-i-switch-language),
+first temporarily change the launch option to _Launch Controller Layout Tool_.
+The drop down menu is located above where you change launch command.
+This will start the thprac launcher, and you can change lauguage under settings.
+Or you can set `LANG` environment variable to `en_US.utf8` for example.
+See below how to set environment variables.
+The launcher setting will take precedence over environment variable.
+
+So, if I wanted to run the game with vpatch and thprac and Brazilian Portuguese translations, the command would look like this:
+
+    thcrap_proton -v -c pt-br.js -p /path/to/thprac.vx.x.x.x.exe -- %command%
```

Comment:
I think it's better to keep the original example intact for people just trying to get the most common use-case working, and then introduce the thprac-specific instructions with another example.